### DISCUSSION TITLE
How to stop the daylight cycle?

### DISCUSSION THREAD

**Discussion by Codestech1** [Minestom/Minestom#1565] [Help] [answered]
I tried it:
`instance.setTimeRate(0);
instance.setTime(-6000);
instance.setTimeUpdate(null);`
but it does not work.

**Comment by KrystilizeNevaDies** [comment, marked as answer]
The answer is revealed in the javadoc:
https://github.com/Minestom/Minestom/blob/d7feed23c8111eef2675d77ddf3dd832905e24fe/src/main/java/net/minestom/server/instance/Instance.java#L403-L414

Use `instance.setTimeRate(0);`

**Reply by Codestech1** [reply]
But i use it and it does not work.

**Reply by Codestech1** [reply]
ok. i removed setTime and setTimeUpdate and it work
